### ISSUE TITLE
Soda Popper stats in tooltip and in-game stats don't match

### ISSUE THREAD

**Issue by conneath** [TF2V/TF2Vintage#29]
while the tooltip in the loadout suggests it builds Hype on hit, it actually builds Hype on moving, like the old Soda Popper before MYM

**Comment by conneath** [comment]
should probably be changed to the default value of tf2v_use_new_sodapopper_fill for now
or should the default value of tf2v_use_new_sodapopper_fill be changed? hmmmmmmmm

**Comment by TF2V** [comment]
Correct, some of the attributes don't change localization right now based on setting, it's why some attributes have strange attribute or inaccurate ones.

**Comment by Deathreus** [comment]
I had a plan for this, but it requires some preamble
